### ISSUE TITLE
itk.Image is not wrapped for input type `itk.SC, int`

### ISSUE THREAD

**Issue by johannes-bp** [KitwareMedical/ITKIOScanco#63]
Hello,

I attempted to use ITKIOScanco to open segmentation masks in *.AIM format. These only contain two values, 0 and 1, and are encoded in signed char (afaik). On trying to open, I received the following error (full stack trace attached below):

> TemplateTypeError: itk.Image is not wrapped for input type `itk.SC, int`.

After some digging, I found a possible solution would be to set the `ComponentType`, as follows:

```
itk.ScancoImageIO.New(ComponentType=itk.CommonEnums.IOComponent_SHORT)
```

however, this change is not passed through, and I still receive the same error. Finally, I was able to mitigate the error by editing `_io_component_type_dict` in the file `itk/support/template_class.py` on Line 179 to read

```
"char": itk.SS,
```

Formerly, this line read

```
"char": itk.SC,
```

which fixed the above mentioned error, but I do not find this solution satisfactory. Unfortunately, I do not have full debugging capabilities on my setup (using Jupyter), so I am unable to follow up where exactly the `ComponentType` is overwritten, however I can verify that after calling `itk.imread()`, the property is set back to `char`, even if I explicitly set it to `short` before.

Any suggestions or help with this issue are strongly appreciated, as I would prefer a "clean" solution to this problem.

For reference, I am using Anaconda3 on Ubuntu 22.04.1 LTS. The installed library versions are

```
itk                       5.2.1.post1
itk-bonemorphometry       1.3.0
itk-core                  5.2.1.post1
itk-filtering             5.2.0
itk-io                    5.2.1.post1
itk-ioscanco              0.9.2
itk-meshtopolydata        0.8.1
itk-numerics              5.2.1.post1
itk-registration          5.2.1.post1
itk-segmentation          5.2.1.post1
itk-texturefeatures       3.5.1
```

Sample files and code examples can be submitted on request.

Full stacktrace: [stacktrace_itkioscanco.txt](https://github.com/KitwareMedical/ITKIOScanco/files/9252170/stacktrace_itkioscanco.txt)

Thank you, and keep up the good work!

Best regards,
Johannes

PS: I use ITKIOScanco regularly to read SCANCO *.AIM volume files in Python, and it works exactly as expected. The problem only occurs with segmentation masks, which are generated from GOBJ-files and often referred to as `*seg.aim`-files by SCANCO.

**Comment by mkuczyns** [comment]
Hi Johannes,

I don't think GOBJ images (signed char) are fully supported in the ITKIOScanco module. However, I've been able to read in Scanco segmentations just fine using this code:

```python
import itk

ImageType = itk.Image[itk.ctype('signed short'), 3]
reader = itk.ImageFileReader[ImageType].New()
imageio = itk.ScancoImageIO.New()
reader.SetImageIO(imageio)
reader.SetFileName('/input/file/path.AIM')
reader.Update()
```

For me, this worked with grayscale AIM images and segmentation images (after conversion from GOBJ to AIM in IPL). I've also been able to run the above code with unsigned char instead of signed short for segmentations with success.

If this doesn't work for you, another option could be to convert your GOBJ mask to AIM in IPL and then run a ```/norm_max``` command to set the AIM to signed short instead of char.
